# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-09-23 16:07+0800\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: app_diet/diet.py:17
msgid "you_are_a_clinical_nutritionist"
msgstr "你是一位临床营养师"

#: app_diet/diet.py:42
msgid "no_diet_record_found_for_{time_of_day}_on_{date}"
msgstr "没有找到{date}{time_of_day}的饮食记录"

#: app_diet/diet.py:58
msgid "no_dietary_records_found_for_keywords_related_to_{time_of_day}_on_{date}"
msgstr "找不到与{time_of_day}有关的{date}的饮食记录"

#: app_diet/diet.py:62
msgid "successfully_deleted_the_diet_record_for_{time_of_day}_on_{date}_with_keywords_{keywords}"
msgstr "成功删除了{date}{time_of_day}的饮食记录，关键字为{keywords}"

#: app_diet/diet.py:139
msgid "total_calories_colon__{kc}kc"
msgstr "总热量：{kc}千卡"

#: app_diet/diet.py:144
msgid "no_valid_records_founds_dot_"
msgstr "未找到有效记录"

#: tests/test_common_llm.py:42
msgid "name"
msgstr "名称"

#: app_diet/diet.py:245
msgid "heat"
msgstr "热量"

#: app_diet/diet.py:246
msgid "carbohydrate"
msgstr "碳水化合物"

#: app_diet/diet.py:247
msgid "fat"
msgstr "脂肪"

#: app_diet/diet.py:248
msgid "protein"
msgstr "蛋白质"

#: app_diet/diet.py:262
msgid "already_exists_{name}"
msgstr "{name} 已经存在"

#: app_diet/diet.py:273
msgid "add_successful_{name}_{calorie}"
msgstr "添加成功 {name} {calorie}"

#: app_diet/diet.py:330
msgid "fuzzy_query_failed"
msgstr "模糊查询失败"

#: app_diet/diet.py:344
msgid "no_food_found_with_the_name_{name}"
msgstr "未找到名称为 {name} 的食物"

#: app_diet/diet.py:385
msgid "today"
msgstr "今天"

#: app_diet/diet.py:386
msgid "yesterday"
msgstr "昨天"

#: app_diet/diet.py:387
msgid "the_day_before_yesterday"
msgstr "前天"

#: app_diet/diet.py:388
msgid "big_day_before_yesterday"
msgstr "大前天"

#: app_diet/diet.py:399
msgid "{month}_month_{day}_day"
msgstr "{month}月{day}日"

#: app_diet/diet.py:410
msgid "spam"
msgstr "午餐肉"

#: app_diet/diet.py:416
msgid "extra_meal_in_the_morning"
msgstr "凌晨加餐"

#: app_diet/diet.py:416
msgid "early_morning"
msgstr "凌晨"

#: app_diet/diet.py:417
msgid "breakfast"
msgstr "早餐"

#: app_diet/diet.py:417
msgid "morning"
msgstr "早上"

#: app_diet/diet.py:418
msgid "morning_extra_meal"
msgstr "上午加餐"

#: tests/test_common_data.py:175
msgid "am"
msgstr "上午"

#: app_diet/diet.py:419
msgid "lunch"
msgstr "午餐"

#: app_diet/diet.py:419
msgid "noon"
msgstr "中午"

#: app_diet/diet.py:420
msgid "extra_meal_in_the_afternoon"
msgstr "下午加餐"

#: app_diet/diet.py:420
msgid "pm"
msgstr "下午"

#: app_diet/diet.py:421
msgid "dinner"
msgstr "晚餐"

#: app_diet/diet.py:421
msgid "dinner2"
msgstr "晚饭"

#: app_diet/diet.py:421
msgid "night"
msgstr "晚上"

#: app_diet/diet.py:422
msgid "evening_extra_meals"
msgstr "晚上加餐"

#: app_diet/diet.py:422
msgid "midnight"
msgstr "半夜"

#: app_diet/diet.py:422
msgid "midnight_snack"
msgstr "夜宵"

#: tests/test_app_diet.py:11
msgid "diet_log_colon__apple_100g_comma__milk_100g"
msgstr "饮食记录 苹果100克，牛奶100克"

#: tests/test_app_diet.py:19
msgid "delete_diet_apple"
msgstr "删除饮食苹果"

#: tests/test_app_diet.py:21
msgid "success"
msgstr "成功"

#: tests/test_app_diet.py:27
msgid "diet_statistics"
msgstr "饮食统计"

#: tests/test_app_diet.py:13
msgid "total_calories"
msgstr "总热量"

#: app_translate/translate.py:19
msgid "you_are_a_middle_school_english_teacher"
msgstr "你是一位中学英语老师"

#: app_translate/translate.py:69
msgid "{}__enter_word_frequency_{}__enter_meaning_{}"
msgstr "{} \n单词频率 {} \n含义 {}"

#: app_translate/translate.py:132
msgid "word_colon__{src}_enter_translation_colon__{dst}_enter_freq_colon__{freq}_enter_count_colon__{times}"
msgstr "单词：{src}\n翻译：{dst}\n频率：{freq}\n次数：{times}"

#: app_translate/views.py:72
msgid "please_enter_questions_or_content_to_be_translated"
msgstr "请输入问题或者待翻译内容"

#: app_translate/views.py:75
msgid "internal_processing_error"
msgstr "内部处理错误"

#: app_translate/dict.py:67
msgid "explanation_colon_"
msgstr "释义："

#: app_translate/word_root.py:19
msgid "root_not_found"
msgstr "未找到词根"

#: app_translate/word_root.py:76
msgid "simple_answer"
msgstr "简单回答"

#: app_translate/word_root.py:19
msgid "root"
msgstr "词根 "

#: app_translate/word_root.py:78
msgid "the_root_of_the_word_is"
msgstr "词根是 "

#: app_translate/word_root.py:81
msgid "affix"
msgstr "词缀 "

#: app_translate/word_root.py:84
msgid "prefix"
msgstr "前缀 "

#: app_translate/translate.py:60
msgid "suffix"
msgstr "后缀 "

#: app_sync/views.py:286
msgid "comparison_complete"
msgstr "对比完成"

#: app_dataforge/misc_tools.py:50
msgid "unsupported_document_type"
msgstr "不支持的文件类型"

#: app_dataforge/feature.py:16
msgid "unclassified"
msgstr "未知类别"

#: app_dataforge/feature.py:165
msgid "to_be_organized"
msgstr "待整理"

#: app_dataforge/feature.py:165
msgid "to-do_list"
msgstr "待办"

#: app_dataforge/feature.py:263
msgid "themes"
msgstr "主题"

#: app_dataforge/views.py:112
msgid "no_update_needed"
msgstr "无需更新"

#: app_dataforge/views.py:118
msgid "uploading_failed"
msgstr "上传失败"

#: app_dataforge/views.py:216
msgid "upgrade_successfully"
msgstr "更新成功"

#: app_dataforge/views.py:218
msgid "update_failed"
msgstr "更新失败"

#: app_dataforge/views.py:292
msgid "unknown_action"
msgstr "未知操作"

#: app_dataforge/entry.py:42
msgid "unknown_type_colon_"
msgstr "未知类型: "

#: app_dataforge/entry.py:90
msgid "save_file_failed_excl_"
msgstr "文件保存失败"

#: app_dataforge/entry.py:135
msgid "add_success"
msgstr "添加成功"

#: app_dataforge/entry.py:140
msgid "add_failed"
msgstr "添加失败"

#: app_dataforge/entry.py:159
msgid "record_successful_comma__type_colon_"
msgstr "记录成功，类型："

#: app_dataforge/entry.py:168
msgid "set_to_pending_comma__type_colon__{ctype}"
msgstr "设为待处理，类型：{ctype}"

#: app_dataforge/entry.py:170
msgid "collection_successful_comma__type_colon__{ctype}"
msgstr "采集成功，类型：{ctype}"

#: app_dataforge/entry.py:174
msgid "storage_successful_comma__found_url_exception_error_colon__{error}"
msgstr "保存成功，找到URL异常错误：{error}"

#: app_dataforge/entry.py:244
msgid "run_failed"
msgstr "运行失败"

#: app_record/record.py:16
msgid "title_colon__enter_{title}_enter__enter_content_enter_{content}"
msgstr "标题：\n{title}\n\n内容\n{content}"

#: app_record/record.py:38
msgid "your_account_currently_does_not_have_export_permissions_dot__please_contact_the_administrator_to_upgrade_your_account_dot_"
msgstr "您的账户暂无导出权限，请联系管理员升级账户"

#: app_record/record.py:47
msgid "website"
msgstr "网址"

#: tests/test_common_data.py:95
msgid "title"
msgstr "标题"

#: tests/support.py:47
msgid "content"
msgstr "内容"

#: app_record/record.py:50
msgid "viewpoint"
msgstr "观点"

#: app_record/record.py:51
msgid "content_category_colon_"
msgstr "内容类别"

#: app_record/record.py:52
msgid "data_type"
msgstr "数据类型"

#: app_dataforge/feature.py:157
msgid "status"
msgstr "状态"

#: app_record/record.py:54
msgid "creation_date"
msgstr "创建日期"

#: app_record/record.py:62
msgid "no_record_found"
msgstr "暂无记录"

#: app_paper/paper_info.py:24
msgid "table_of_contents"
msgstr "目录"

#: app_paper/paper_info.py:31
msgid "abstract"
msgstr "摘要"

#: app_paper/views.py:43
msgid "method_not_supported_colon_"
msgstr "不支持方法:"

#: app_paper/ptools.py:59
msgid "natural_language_processing"
msgstr "自然语言处理"

#: app_paper/ptools.py:60
msgid "medicine"
msgstr "医学"

#: app_paper/ptools.py:61
msgid "computer_vision"
msgstr "计算机视觉"

#: tests/test_common_audio.py:6
msgid "audio"
msgstr "音频"

#: app_paper/ptools.py:63
msgid "video"
msgstr "视频"

#: app_paper/ptools.py:64
msgid "reinforcement_learning_(rl)"
msgstr "强化学习"

#: app_paper/ptools.py:65
msgid "knowledge_map"
msgstr "知识图谱"

#: app_paper/ptools.py:66
msgid "machine_learning"
msgstr "机器学习"

#: app_paper/ptools.py:109
msgid "have_read"
msgstr "已读"

#: app_paper/ptools.py:110
msgid "unread"
msgstr "未读"

#: app_paper/ptools.py:110
msgid "to_read"
msgstr "待读"

#: app_paper/ptools.py:110
msgid "unseen"
msgstr "未看"

#: app_paper/ptools.py:111
msgid "reading_section"
msgstr "读部分"

#: app_paper/ptools.py:113
msgid "watching"
msgstr "正在看"

#: app_paper/ptools.py:114
msgid "watch_a_little"
msgstr "看一点"

#: app_dataforge/views.py:292
msgid "unknown"
msgstr "未知"

#: app_translate/translate.py:80
msgid "chinese"
msgstr "中文"

#: app_paper/ptools.py:384
msgid "paper_reading"
msgstr "论文阅读"

#: app_paper/ptools.py:394
msgid "english_title_colon__{title}"
msgstr "英文名称：{title}"

#: app_paper/ptools.py:396
msgid "english_name_colon_"
msgstr "英文名称: "

#: app_paper/ptools.py:398
msgid "chinese_name_colon__{title_zh}"
msgstr "中文名称：{title_zh}"

#: app_paper/ptools.py:398
msgid "chinese_name_colon_"
msgstr "中文名称:"

#: app_paper/ptools.py:402
msgid "link_colon__{url}"
msgstr "链接: {url}"

#: app_paper/ptools.py:402
msgid "link_colon_"
msgstr "链接:"

#: app_paper/ptools.py:405
msgid "code_colon_"
msgstr "代码:"

#: app_paper/ptools.py:407
msgid "author_colon__{author}"
msgstr "作者: {author}"

#: app_paper/ptools.py:407
msgid "author_colon_"
msgstr "作者: "

#: app_paper/ptools.py:412
msgid "institution_colon__{institution}"
msgstr "机构：{institution}"

#: app_paper/ptools.py:412
msgid "institution_colon_"
msgstr "机构: "

#: app_paper/ptools.py:418
msgid "date_colon__{published_date}"
msgstr "日期：{published_date}"

#: app_paper/ptools.py:418
msgid "date_colon_"
msgstr "日期: "

#: app_paper/ptools.py:423
msgid "citations_colon__{citations}"
msgstr "引文数量：{citations}"

#: app_paper/ptools.py:425
msgid "number_of_citations_colon_"
msgstr "引用次数: "

#: app_paper/ptools.py:429
msgid "objective_colon__{purpose}"
msgstr "目标：{purpose}"

#: app_paper/ptools.py:431
msgid "target_colon_"
msgstr "目标: "

#: app_paper/ptools.py:433
msgid "method_colon__{method}"
msgstr "方法：{method}"

#: app_paper/ptools.py:433
msgid "method_colon_"
msgstr "方法: "

#: app_paper/ptools.py:437
msgid "conclusion_colon__{result}"
msgstr "结论：{result}"

#: app_paper/ptools.py:437
msgid "conclusion_colon_"
msgstr "结论:"

#: app_paper/ptools.py:574
msgid "empty_contents"
msgstr "内容为空"

#: app_paper/ptools.py:602
msgid "original_text"
msgstr "原文"

#: app_paper/ptools.py:602
msgid "polished_text"
msgstr "润色后的文本"

#: app_paper/ptools.py:602
msgid "translate_text"
msgstr "翻译后的文本"

#: backend/common/user/resource.py:70
msgid "speech_synthesis"
msgstr "语音合成"

#: backend/common/user/resource.py:70
msgid "language_model"
msgstr "语言模型"

#: backend/common/user/resource.py:72
msgid "current_day"
msgstr "本日"

#: backend/common/user/resource.py:73
msgid "current_week"
msgstr "本周"

#: backend/common/user/resource.py:74
msgid "this_month"
msgstr "本月"

#: tests/test_app_diet.py:13
msgid "total"
msgstr "总计"

#: backend/common/user/resource.py:79
msgid "{}_usage"
msgstr "{} 用量"

#: tests/test_app_user_manager.py:30
msgid "username_already_exists"
msgstr "用户已存在"

#: backend/common/user/views.py:55
msgid "account_created__excl_"
msgstr "注册成功"

#: backend/common/user/views.py:57
msgid "registration_failed"
msgstr "注册失败"

#: backend/common/user/views.py:59
msgid "user_or_password_are_empty_dot_"
msgstr "用户名或密码为空"

#: tests/test_app_user_manager.py:42
msgid "user_does_not_exist"
msgstr "用户不存在"

#: tests/test_app_user_manager.py:64
msgid "original_password_error"
msgstr "原密码错误"

#: tests/test_app_user_manager.py:57
msgid "successfully_set"
msgstr "设置成功"

#: backend/common/user/views.py:78
msgid "setup_failed"
msgstr "设置失败"

#: backend/common/user/views.py:80
msgid "the_new_password_is_blank"
msgstr "新密码为空"

#: backend/common/user/views.py:116
msgid "user_level_colon__{level_desc}"
msgstr "用户等级: {level_desc}"

#: backend/common/user/views.py:157
msgid "settings_were_applied_successfully"
msgstr "设置成功"

#: backend/common/user/user.py:37
msgid "chat_prompt_default"
msgstr "请简单回答"

#: backend/common/user/user.py:147
msgid "supports_ocr_colon_"
msgstr "支持OCR:"

#: app_diet/diet.py:386
msgid "yes"
msgstr "是"

#: backend/common/user/user.py:147
msgid "denied"
msgstr "否"

#: backend/common/user/user.py:148
msgid "save_data_colon_"
msgstr "保存数据:"

#: backend/common/user/user.py:150
msgid "parsing_the_paper_colon_"
msgstr "解析论文:"

#: backend/common/user/user.py:153
msgid "translation_book_colon_"
msgstr "翻译书籍:"

#: backend/common/user/user.py:157
msgid "model_rate_limit_colon__{limit_llm_day_formatted}_tokens/day"
msgstr "模型限制：{limit_llm_day_formatted} 个token/天"

#: backend/common/user/user.py:162
msgid "speech_synthesis_rate_limit_colon__{limit_tts_day}_tokens/day"
msgstr "语音合成限制：{limit_tts_day} 个token/天"

#: backend/common/user/user.py:167
msgid "speech_synthesis_rate_limit_colon__{tokens}_tokens/attempt"
msgstr "语音合成限制：每次尝试 {tokens} 个token"

#: backend/common/user/user.py:172
msgid "recording_export_range_colon__{limit_export_record_day}_days"
msgstr "记录导出范围：{limit_export_record_day} 天"

#: backend/common/user/user.py:191
msgid "te_ac_settings"
msgstr "临时用户"

#: app_dataforge/feature.py:152
msgid "normal"
msgstr "普通用户"

#: backend/common/user/user.py:195
msgid "vip_user"
msgstr "VIP用户"

#: backend/common/user/user.py:197
msgid "administrator"
msgstr "管理员"

#: backend/common/user/user.py:198
msgid "unknown_level"
msgstr "未知级别"

#: backend/common/utils/regular_tools.py:26
msgid "content_not_found"
msgstr "无内容"

#: backend/common/utils/web_tools.py:17
msgid "unknown_title"
msgstr "未知标题"

#: backend/common/utils/web_tools.py:34
msgid "empty_file_content"
msgstr "文件内容为空"

#: backend/common/utils/web_tools.py:112
msgid "parsing_content_is_not_supported_at_this_time"
msgstr "暂不支持解析内容"

#: backend/common/utils/web_tools.py:139
msgid "failure_reading_file"
msgstr "读取文件失败"

#: backend/common/utils/web_tools.py:247
msgid "body"
msgstr "正文"

#: backend/common/speech/tts_xunfei.py:62
msgid "synthesis_complete"
msgstr "合成完成"

#: backend/common/speech/asr_tools.py:52
msgid "recognition_failure"
msgstr "识别失败"

#: backend/common/speech/asr_tools.py:75
msgid "asr_successful"
msgstr "识别成功"

#: backend/common/speech/tts_tools.py:282
msgid "assembled"
msgstr "合成成功"

#: backend/common/speech/tts_tools.py:285
msgid "custom_synthesis_failed"
msgstr "定制合成失败"

#: backend/common/speech/tts_tools.py:287
msgid "synthesis_failed"
msgstr "合成失败"

#: backend/common/speech/tts_tools.py:359
msgid "the_current_user_already_has_a_task_in_progress"
msgstr "当前用户已有任务在执行"

#: backend/common/speech/tts_tools.py:361
msgid "the_current_task_list_is_full_comma__please_try_again_later"
msgstr "当前任务列表已满，请稍后再试"

#: backend/common/speech/tts_tools.py:368
msgid "the_task_has_been_added__excl_"
msgstr "任务已添加"

#: backend/common/speech/tts_tools.py:376
msgid "task_removed"
msgstr "任务已移除"

#: backend/common/speech/tts_tools.py:377
msgid "quest_does_not_exist"
msgstr "任务不存在"

#: backend/common/speech/tts_tools.py:460
msgid "in_the_queue_comma__there_are_still_{prev_task_n}_tasks_ahead"
msgstr "队列中仍有 {prev_task_n} 个任务在前"

#: backend/common/speech/tts_tools.py:472
msgid "currently_synthesizing_comma__{_colon__dot_1%}_completed"
msgstr "正在合成中，已完成 {:.1%}"

#: backend/common/speech/tts_tools.py:479
msgid "no_synthesized_audio_found"
msgstr "没有找到合成音频"

#: backend/common/speech/tts.py:68
msgid "the_daily_limit_of_synthesized_characters_has_been_reached_{limit_tts_day}"
msgstr "已达到每日合成字符限制 {limit_tts_day}"

#: backend/common/speech/tts.py:95
msgid "the_content_is_relatively_long_{length}_characters_comma__and_it_will_take_about_{time}_minutes_to_synthesize_dot__if_you_do_not_wish_to_continue_converting_comma__please_reply_colon__stop"
msgstr "内容相对较长，有 {length} 个字符，预计需要 {time} 分钟来合成。如果你不想继续转换，请回复: 停止语音合成"

#: backend/common/speech/tts.py:112
msgid "no_content_found"
msgstr "没有找到相应内容"

#: backend/common/speech/tts.py:135
msgid "set_the_engine_to_colon__{base_engine_setting}"
msgstr "将引擎设置为：{base_engine_setting}"

#: backend/common/speech/tts.py:143
msgid "set_engine_to_custom_colon__{base_engine_setting}"
msgstr "设置引擎为自定义: {base_engine_setting}"

#: backend/common/speech/tts.py:146
msgid "engine_not_found_colon__{base_engine_setting}"
msgstr "未找到引擎：{base_engine_setting}"

#: backend/common/speech/tts.py:160
msgid "microsoft"
msgstr "微软"

#: data/grpc_asyncio.py:2
msgid "google"
msgstr "谷歌"

#: backend/common/speech/tts_xunfei.py:28
msgid "xunfei"
msgstr "讯飞"

#: backend/common/speech/tts.py:164
msgid "customization"
msgstr "定制"

#: app_translate/urls.py:3
msgid "default"
msgstr "默认"

#: backend/common/speech/tts.py:166
msgid "storytelling_fairy"
msgstr "讲故事的小仙女"

#: backend/common/speech/tts.py:168
msgid "book_review_master"
msgstr "评书大师"

#: backend/common/speech/tts.py:169
msgid "ghost_story_expert"
msgstr "鬼故事专家"

#: backend/common/speech/tts.py:170
msgid "romance_novel"
msgstr "言情小说"

#: backend/common/speech/tts.py:171
msgid "documentary_commentary"
msgstr "记录片解说"

#: backend/common/speech/tts.py:172
msgid "clear_female_voice"
msgstr "清澈女声"

#: backend/common/speech/tts.py:173
msgid "deep_male_voice"
msgstr "深沉男声"

#: backend/common/speech/tts.py:176
msgid "gentle_male_voice"
msgstr "温和男声"

#: backend/common/files/utils_file.py:193
msgid "total_{len_files}_files"
msgstr "总计 {len_files} 个文件"

#: backend/common/files/utils_file.py:194
msgid "total_{total}_tokens"
msgstr "总共 {total} 个代币"

#: backend/common/files/utils_file.py:196
msgid "embedding_costs_about_{cost}_yuan"
msgstr "嵌入花费约 {cost} 元"

#: backend/common/parser/doc_parser.py:13
msgid "successfully_converted_{path_in}_to_{path_out}"
msgstr "成功将 {path_in} 转换为 {path_out}"

#: backend/common/parser/block.py:176
msgid "appendices"
msgstr "附录"

#: backend/common/parser/block.py:177
msgid "attachment"
msgstr "附件"

#: backend/common/parser/block.py:178
msgid "foreword"
msgstr "序言"

#: backend/common/parser/block.py:179
msgid "given_that_colon_"
msgstr "鉴于:"

#: backend/common/parser/block.py:180
msgid "in_view_of_the_terms_colon_"
msgstr "鉴于条款："

#: backend/common/parser/block.py:186
msgid "(no_text)"
msgstr "无正文"

#: backend/common/parser/block.py:186
msgid "signing_page"
msgstr "签字页"

#: backend/common/parser/block.py:186
msgid "page_for_signing"
msgstr "签署页"

#: backend/common/llm/llm_hub.py:41
msgid "the_maximum_number_of_words_called_today_has_been_reached"
msgstr "今日调用字数已达上限"

#: backend/common/llm/llm_hub.py:85
msgid "call_failed"
msgstr "调用失败"

#: backend/common/llm/llm_hub.py:127
msgid "usage_limit_colon__{limit_llm_day}_comma__used_colon__{used_llm_count}"
msgstr "使用限制：{limit_llm_day}，已用：{used_llm_count}"

#: backend/common/llm/llm_tools.py:145
msgid "gemini_is_temporarily_unavailable"
msgstr "gemini暂时无法使用"

#: app_bm_syncex/views.py:49
msgid "data_already_exists"
msgstr "数据已存在"

#: app_message/function.py:31
msgid "chat_call_failed"
msgstr "chat调用失败"

#: app_message/views.py:57
msgid "backend_processing_failed"
msgstr "后端处理失败"

#: app_message/views.py:68
msgid "please_sign_up_or_log_in_first"
msgstr "请先注册或登录"

#: app_message/user_manager.py:11
msgid "please_register_or_log_in_first_comma___enter_formatted_as_colon__register_username_xxx_comma__password_xxx__enter_or_log_in_username_xxx_comma__password_xxx"
msgstr "请先注册或登录，\n形如：注册 用户名xxx, 密码xxx \n或 登录 用户名xxx, 密码xxx"

#: app_message/user_manager.py:38
msgid "empty_username"
msgstr "用户名为空"

#: tests/test_app_user_manager.py:64
msgid "old_password_is_empty"
msgstr "原密码为空"

#: app_message/user_manager.py:44
msgid "old_and_new_passwords_are_identical_dot_"
msgstr "新旧密码相同"

#: app_message/command.py:100
msgid "command_not_found"
msgstr "找不到命令"

#: tests/test_app_user_manager.py:13
msgid "login_colon__xieyan_comma__1234567"
msgstr "登录：xieyan, 1234567"

#: tests/test_app_user_manager.py:21
msgid "registration_colon__xieyan_comma__1234567"
msgstr "注册：xieyan, 1234567"

#: tests/test_app_user_manager.py:36
msgid "change_password_colon__xieyan_comma__old_password_1234567_new_password_123456"
msgstr "修改密码：xieyan, 原密码 1234567 新密码 123456"

#: tests/test_app_user_manager.py:60
msgid "change_password_colon__xieyan_comma__2314"
msgstr "修改密码：xieyan, 2314"

#: tests/test_app_user_manager.py:72
msgid "when_to_use_a_beaker_and_when_to_use_a_test_tube_in_chemistry"
msgstr "化学什么时候用烧杯，什么时候用试管"

#: tests/test_app_record.py:12
msgid "record_exercise_running_5_kilometers"
msgstr "记录 锻炼 跑步5公里"

#: app_message/message.py:63
msgid "please_choose_colon_\n"
msgstr "请选择：\n"

#: app_message/message.py:78
msgid "no_urls_dot_"
msgstr "没有网址"

#: app_message/message.py:112
msgid "failed_to_fetch_webpages"
msgstr "提取网页失败"

#: app_message/message.py:141
msgid "page_not_found"
msgstr "没有找到网页"

#: app_message/message.py:869
msgid "failed_to_process_information"
msgstr "处理信息失败"

#: app_message/message.py:907
msgid "the_file_type_is_not_valid_or_not_supported"
msgstr "文件类型不支持"

#: app_message/message.py:935
msgid "no_content_found_1727252424"
msgstr "未找到相应内容"

#: app_message/message.py:235
msgid "please_upload_or_share_a_file_first"
msgstr "请先上传或分享文件"

#: app_message/message.py:260
msgid "file_collection_failed_colon__file_not_found"
msgstr "文件收藏失败：未找到文件"

#: app_message/message.py:327
msgid "failed_to_fetch_files"
msgstr "获取文件失败"

#: app_message/message.py:177
msgid "please_enter_the_url_or_share_the_page_with_me"
msgstr "请输入网址或者分享网页给我"

#: app_message/message.py:369
msgid "files_larger_than_50mb_are_currently_not_supported_for_recognition"
msgstr "文件大于50M，暂不支持识别"

#: app_message/message.py:342
msgid "please_enter_what_you're_looking_for"
msgstr "请输入待查找内容"

#: app_message/message.py:431
msgid "please_enter_the_record_content"
msgstr "请输入记录内容"

#: app_message/message.py:812
msgid "translation_failed"
msgstr "翻译失败"

#: app_message/message.py:841
msgid "nothing_entered"
msgstr "没有输入"

#: app_message/message.py:843
msgid "unrecognized_command"
msgstr "不识别的命令"

#: app_message/message.py:883
msgid "no_optional_commands"
msgstr "没有可选命令"

#: app_message/message.py:886
msgid "not_a_number"
msgstr "不是数字"

